driver.sh

```diff
@@ -70,6 +70,17 @@ setup_variables() {
       qemu_cmdline=( -drive "file=images/x86_64/rootfs.ext4,format=raw,if=ide"
                      -append "console=ttyS0 root=/dev/sda" ) ;;
 
+    "ppc64le")
+      config=powernv_defconfig
+      image_name=zImage.epapr
+      qemu="qemu-system-ppc64"
+      qemu_cmdline=( -M powernv -m 3G -nographic
+                     -device ipmi-bmc-sim,id=bmc0 -device isa-ipmi-bt,bmc=bmc0,irq=10
```

Comment:
Would you mind putting these on separate lines if the current line L78 runs over 80 chars? (I can't tell from GH UI).